solution/backend/supplemental_content/views.py

```diff
@@ -69,7 +69,7 @@ def get(self, *args, **kwargs):
             locations__title=title,
             locations__part=part,
         )
-        if len(section_list) > 0 or len(subpart_list) > 0 and len(subjgrp_list) > 0:
+        if len(section_list) > 0 or len(subpart_list) > 0 or len(subjgrp_list) > 0:
```

Comment:
this bug made it to prod but this will fix it.

Reply:
looks fine.